packages/localize-tools/src/modes/transform.ts

```diff
@@ -448,7 +448,7 @@ class Transformer {
           'Internal error: string literal cannot be html-tagged'
         );
       }
-      return template;
+      return newTemplate;
```

Comment:
Oops! Do the tests in this PR fail if this is switched back to `template`? If not, is there a test we could add that would cover it?

Reply:
Upon taking a closer look now, it seems we never enter this code block if a translation does occur because then `newTemplate` would always be a `ts.TemplateLiteral`. The only time we enter this block is when we don't have to translate, i.e. generating original locale's version of the source, in which case `newTemplate` would not have changed from it's original assignment from `template` here:
https://github.com/lit/lit/blob/f78b4cc0285c4df5248324367453a5c4a0f3d1d6/packages/localize-tools/src/modes/transform.ts#L335
So.. it was technically an oversight but of no consequence, hence none of our existing tests failed. For readability's sake I think keeping `newTemplate` here still makes sense.

Reply:
sgtm